scripts/segmentation/test.py

```diff
@@ -68,14 +68,14 @@ def test(args):
             im_paths = dsts
             predicts = evaluator.parallel_forward(data)
             for predict, impath in zip(predicts, im_paths):
-                predict = mx.nd.squeeze(mx.nd.argmax(predict[0], 1)).asnumpy() + \
-                    testset.pred_offset
+                predict = mx.nd.squeeze(mx.nd.argmax(predict, 1), axis=0).\
```

Comment:
This will break existing segmentation model, which outputs a list by default.

Reply:
What are the elements in the list `predict`?
The shape of `predict[0]` is (num_cls, height, weight)
I think `predicts` is a list of `NDArray`, and `predict` is a `NDArray` whose `batch_size` is 1.

Reply:
predicts is a list of list of NDArray

Reply:
Thank you! I will check it.

Reply:
@zhanghang1989 
Hi, I found the type of `predicts` is `tuple`, and the type of `predict` is `NDArray`.

Test script: `python test.py --dataset pascal_voc --model-zoo psp_resnet101_voc`